lib/views/list.js

```diff
@@ -65,9 +68,14 @@ class ListView extends React.Component {
   renderListItems() {
     debug('render list items');
     return this.props.items.map(item => {
+      debug('render item', item.id);
       return <ListItem
         {...item}
         onLongPress={this.onItemLongPress}
+        onGestureStart={this.onItemGestureStart.bind(this)}
+        onGestureEnd={this.onItemGestureEnd.bind(this)}
+        onSwiped={this.onItemSwiped.bind(this)}
+        swipable={true}
```

Comment:
I added it in case we wanted to land this feature behind a flag. If we're happy to go ahead I'll remove the flag.